app/views/homes/index.html.erb

```diff
@@ -28,9 +28,10 @@
     <p class="card-text"><%= link_to truncate(post.description), post %></p>
     <div class="img-div"><%= image_tag post.image, width:"500px", class:"img-fluid" if post.image.attached?%></div>
   </div>
-  <div class="card-footer text-muted">
-    <a><i class="fa fa-thumbs-up"> 7 </i></a></a>
-    <a <i class="fa fa-comment-o"> <%= link_to " comments", new_post_path %> </i></a>
+  <div class="card-footer text-muted like-and-comments">

```

Comment:
- [ ] With our current implementation a user can likes a post multiple time. Let's check here if the user is already likes, to display a dislike button. Also don't forget the wrap this view logic in a helper method.

- [ ] **[OPTIONAL]** The previous step handles likes duplication from UI perspective, but to ensure our data integrity, we can add validation to forbid a user to add multiple likes. We need to validate that [user_id, post_id] in likes table is always unique. We can use [`scope`](https://guides.rubyonrails.org/active_record_validations.html#uniqueness) to achieve this in our likes model.